matching-notification-resources/src/main/resources/resources/uicomponents/matchingNotification/matchesTable.js

```diff
@@ -372,8 +380,20 @@ var PhenoTips = (function (PhenoTips) {
             var isNotifiedMatch = match.notified && this._filterValues.notified.notified || !match.notified && this._filterValues.notified.unnotified;
             var hasScoreMatch = match.score >= this._filterValues.score.score
                              && match.phenotypicScore >= this._filterValues.score.phenotypicScore
-                             && match.genotypicScore >= this._filterValues.score.genotypicScore;
-            return hasExternalIdMatch && hasEmailMatch && hasGeneSymbolMatch && hasAccessTypeMath && hasOwnershipMatch
+
+            // returns true if one of the records in match is local and owned my user and is solved
+            var matchHasOwnSolvedCase = function(match) {
+                    if (match.reference.serverId == "" && match.reference.ownership["userIsOwner"] && match.reference.pubmedIDs && match.reference.pubmedIDs.size() > 0) {
```

Comment:
I feel that the check shouldn't be on `pubmedIds.size > 0` (which is done in too many places), but on a simple `solved` boolean.